.github/workflows/ci-test.yml

```diff
@@ -0,0 +1,110 @@
+#
```

Comment:
Can you leave a comment or TODO that this will be deleted in the future? (Correct?)

Reply:
It's correct that this test is likely to be removed after the migration (but that's true for most of the 16 existing workflows)

Reply:
Actually this one is not entirely obsolete (there's at least yamllint and actionlint in there). I will leave this as is: We'll need to review and clean up the workflows afterwards but that is listed in the migration plan.